### PR TITLE
Add sublabel for Configuration+(s)

### PR DESCRIPTION
**Pull request by lasers**
`Show Hidden Files and Folders` belongs in Configuration or User Interface?

### CODE REVIEW

File: intl/msg_hash_chs.c [hunk 1]
```diff
@@ -3242,6 +3242,10 @@ const char *msg_hash_to_str_chs(enum msg_hash_enums msg)
          return "启用或者禁止蓝牙。";
       case MENU_ENUM_SUBLABEL_CONFIG_SAVE_ON_EXIT:
          return "程序将在退出时保存修改到配置文件。";
+      case MENU_ENUM_SUBLABEL_CONFIGURATION_SETTINGS:
+         return "Change default settings for configuration, content override, core override, and remap files.";
```
Comment (twinaphex) [inline intl/msg_hash_chs.c]: This is way too specific, too much information overload.

File: intl/msg_hash_chs.c [hunk 2]
```diff
@@ -3242,6 +3242,10 @@ const char *msg_hash_to_str_chs(enum msg_hash_enums msg)
...
+      case MENU_ENUM_SUBLABEL_CONFIGURATIONS_LIST:
+         return "Load, save, new.";
```
Comment (twinaphex) [inline intl/msg_hash_chs.c]: This doesnt mean anything to a regular user.

**Comment by lasers** [comment]
(Current)
```
Change default settings for configuration, content
override, core override, and remaps files.
```

(Possible Option) Both are same. not sure about wrap w/o testing.
```
Change default settings for configuration, override,
and remaps files.   <--- somewhat short on 2nd line.
```
```
Change default settings for configuration,
override, and remaps files.
```

(Simple)
```
Change default settings for configuration files.
```
This may be just a case of initial information overload
then quick glances in the future like `Information` or `Onscreen Display`.

Which option do you want to opt for?

Also, change `Load, save, new.` to what?
`Load, save, and create new configuration` ?

**Comment by hizzlekizzle** [comment]
how about:
Determine how RetroArch handles configuration files
and
Create and manage configuration files

**Comment by lasers** [comment]
`Manage configuration files.` could work.
`Manage configuration file.` could work.

Is it just one file or multiple? I notice `Save Current Configuration` will make a backup file if retroarch.cfg isn't writeable. 

I think we can't use RetroArch in labels/sublabels because of Lakka. I cleaned up `RetroArch Tour` --> `Help` before it went in.

@hizzlekizzle: Which option do you want to opt for? Possible or Simple?

**Comment by hizzlekizzle** [comment]
I'm of the opinion that sublabels should be simple and the expanded info (right-shift hotkey, IIRC?) should be the complex, verbose bit.

If we want to avoid using RetroArch for Lakka's sake, we can always just switch to the loathsome passive voice:
Determine how configuration files are loaded and prioritized

**Comment by twinaphex** [comment]
Sublabels should indeed be simple and the expanded info the more complex, verbose ones, exactly like @hizzlekizzle suggested.

**Pull request merged**